src/Microsoft.OData.Core/UriParser/SyntacticAst/ExpandTermToken.cs

```diff
@@ -104,6 +109,36 @@ public ExpandTermToken(PathSegmentToken pathToNavigationProp, SelectToken select
         /// <param name="selectOption">the select option for this expand term</param>
         /// <param name="expandOption">the expand option for this expand term</param>
         public ExpandTermToken(PathSegmentToken pathToNavigationProp, QueryToken filterOption, IEnumerable<OrderByToken> orderByOptions, long? topOption, long? skipOption, bool? countQueryOption, long? levelsOption, QueryToken searchOption, SelectToken selectOption, ExpandToken expandOption)
+            : this(pathToNavigationProp, filterOption, orderByOptions, topOption, skipOption, countQueryOption, levelsOption, searchOption, selectOption, null, expandOption)
+        {
+        }
+
+        /// <summary>
+        /// Create an expand term token
+        /// </summary>
+        /// <param name="pathToNavigationProp">the nav prop for this expand term</param>
+        /// <param name="filterOption">the filter option for this expand term</param>
+        /// <param name="orderByOptions">the orderby options for this expand term</param>
+        /// <param name="topOption">the top option for this expand term</param>
+        /// <param name="skipOption">the skip option for this expand term</param>
+        /// <param name="countQueryOption">the query count option for this expand term</param>
+        /// <param name="levelsOption">the levels option for this expand term</param>
+        /// <param name="searchOption">the search option for this expand term</param>
+        /// <param name="selectOption">the select option for this expand term</param>
+        /// <param name="computeOptions">the compute option for this expand term.</param>
+        /// <param name="expandOption">the expand option for this expand term</param>
+        public ExpandTermToken(
+            PathSegmentToken pathToNavigationProp,
+            QueryToken filterOption,
+            IEnumerable<OrderByToken> orderByOptions,
+            long? topOption,
+            long? skipOption,
+            bool? countQueryOption,
+            long? levelsOption,
+            QueryToken searchOption,
+            SelectToken selectOption,
+            IEnumerable<ComputeToken> computeOptions,

```

Comment:
Why did you add computeOptions *before* expandOption? Generally better for overloads to build on previous parameter ordering.